examples/identity/src/main.sw

```diff
@@ -48,7 +48,7 @@ impl IdentityExample for Contract {
 
         // ANCHOR: different_executions
         match my_identity {
-            Identity::Address(identity) => transfer_to_output(amount, token_id, identity),
+            Identity::Address(identity) => transfer_to_address(amount, token_id, identity),
```

Comment:
I find this example slightly misleading. the inner value name below would be better as just `a` or `addr` or similar. Having the value named `identity` could be confused with the `Identity` type.
```
Identity::Address(identity) => transfer_to_address(amount, token_id, identity),
```